pysph/base/opencl.py

```diff
@@ -35,6 +42,31 @@ def set_queue(q):
     _queue = q
 
 
+def profile(name, event):
+    global _profile_info
+    event.wait()
+    time = (event.profile.end - event.profile.start) * 1e-9
+    if name in _profile_info:
+        _profile_info[name] += time
+    else:
+        _profile_info[name] = time
+
+
+def profile_info():
```

Comment:
Don't like this, name, merge the `profile_info` and `print_profile` into one function saying `print_profile()`.